src/cpp/src/visual_language/llava_next_video/classes.cpp

```diff
@@ -58,7 +215,18 @@ clip_image_f32 preprocess_clip_image_llava_next_video(const clip_image_u8& image
 VisionEncoderLLaVANextVideo::VisionEncoderLLaVANextVideo(
     const std::filesystem::path& model_dir,
     const std::string& device,
-    const ov::AnyMap properties) : VisionEncoderLLaVANext(model_dir, device, properties) {
+    const ov::AnyMap properties) : VisionEncoderLLaVANext(model_dir, device, properties),
+        use_ov_preprocess(can_use_ov_video_preprocess()) {
+    if (use_ov_preprocess) {
+        auto preprocess_model = create_video_preprocess_model(m_processor_config);
+        auto compiled_preprocess = utils::singleton_core().compile_model(preprocess_model, device, properties);
+        m_ireq_queue_preprocess = std::make_unique<CircularBufferQueue<ov::InferRequest>>(
+            compiled_preprocess.get_property(ov::optimal_number_of_infer_requests),
+            [&compiled_preprocess]() -> ov::InferRequest {
+                return compiled_preprocess.create_infer_request();
+            });
```

Comment:
In similar PRs we had a pattern that preprocessing was patched into vision encoder model ([see example](https://github.com/openvinotoolkit/openvino.genai/pull/3063/files#diff-1a500ce044bca05c9b72f55d383849cfe36d78e60cef128c081cc46c3a0c25e1R679)).
I would follow the same approach if it does not contradict/interfere with image input preprocessing.